client/src/test/java/org/threadly/litesockets/client/http/TestUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.threadly.litesockets.client.http;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class TestUtils {
```

Comment:
This seems like it might be useful in the top level litesockets project.

Not just because we need it for unit testing, but it seems like a common pattern for unit testing network applications in general (so for others building projects based off litesockets).